pkg/config/validation/validation.go

```diff
@@ -1936,6 +1941,30 @@ func ValidateControlPlaneAuthPolicy(policy meshconfig.AuthenticationPolicy) erro
 	return fmt.Errorf("unrecognized control plane auth policy %q", policy)
 }
 
+func validatePolicyTargetReference(selector *type_beta.WorkloadSelector, targetRef *type_beta.PolicyTargetReference) (v Validation) {
+	if targetRef == nil {
+		return
+	}
+	if selector != nil && targetRef != nil {
+		v = appendErrorf(v, "policyTargetReference and selector cannot both be set")
```

Comment:
```suggestion
		v = appendErrorf(v, "targetRef and selector cannot both be set")
```

nit: users use the YAML api where its targetRef